doc/Security.xml

```diff
@@ -4867,6 +4875,55 @@ CSeq: 2
 User-Agent: ./onvifClient
 Accept: application/sdp]]></programlisting>
   </appendix>
+
+  <appendix xml:id= "b14_55m_51d">
+    <title>Cipher Reference</title>
```

Comment:
Change title to "Cipher Reference (Informative)".

Reply:
@HansBusch updated as requested.